push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/MessageWithTokens.java

```diff
@@ -0,0 +1,41 @@
+package org.jboss.aerogear.unifiedpush.message;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jboss.aerogear.unifiedpush.api.PushMessageInformation;
+import org.jboss.aerogear.unifiedpush.api.Variant;
+
+public class MessageWithTokens implements Serializable {
```

Comment:
It is a single-purpose serializable crate for specific JMS payload (didn't find any suitable existing type).

On later occasion we may also want to optimize how it gets serialized/deserialized.

Adding javadoc.